mos-platform/common/include/string.h

```diff
@@ -18,11 +21,11 @@ char *strchr(const char *s, int c);
 
 int strcmp(const char *s1, const char *s2);
 
-char* strcpy(char *restrict s1, const char *restrict s2);
+char *strcpy(char * __RESTRICT s1, const char * __RESTRICT s2);
```

Comment:
Try `__restrict__`; this should work in Clang for both C and C++.

Reply:
Good to know! Thank you!